serviceaccount/serviceaccount_test.go

```diff
@@ -147,3 +148,77 @@ func TestDeleteUsingDeploymentUUID(t *testing.T) {
 `
 	assert.Equal(t, buf.String(), expectedOut)
 }
+
+func TestGetDeploymentServiceAccount(t *testing.T) {
+	testUtil.InitTestConfig()
+	okResponse := `
+	{
+		"data": {
+		  "deploymentServiceAccounts": [
+		    {
+		      "id": "ckqvfa2cu1468rn9hnr0bqqfk",
+		      "apiKey": "658b304f36eaaf19860a6d9eb73f7d8a",
+		      "label": "yooo can u see me test",
+		      "category": "",
+		      "entityType": "DEPLOYMENT",
+		      "entityUuid": null,
+		      "active": true,
+		      "createdAt": "2021-07-08T21:28:57.966Z",
+		      "updatedAt": "2021-07-08T21:28:57.967Z",
+		      "lastUsedAt": null
+		    }
+		  ]
+		}
+	      }`
+	client := testUtil.NewTestClient(func(req *http.Request) *http.Response {
+		return &http.Response{
+			StatusCode: 200,
+			Body:       ioutil.NopCloser(bytes.NewBufferString(okResponse)),
+			Header:     make(http.Header),
+		}
+	})
+	api := houston.NewHoustonClient(client)
+	deploymentUuid := "ckqvf9spa1189rn9hbh5h439u"
+	buf := new(bytes.Buffer)
+	err := GetDeploymentServiceAccounts(deploymentUuid, api, buf)
+	assert.NoError(t, err)
+	expectedOut := ` yooo can u see me test                  ckqvfa2cu1468rn9hnr0bqqfk     658b304f36eaaf19860a6d9eb73f7d8a`
+	assert.Contains(t, buf.String(), expectedOut)
+}
+
+func TestGetWorkspaceServiceAccount(t *testing.T) {
+	testUtil.InitTestConfig()
+	okResponse := `
+	{
```

Comment:
we are trying to keep those jsons with 2 spaces formating.

Reply:
updated spacing in added json